### PR TITLE
Task detect errors and rerun task

### LINKED ISSUE #108: Rerun task
- [x] Rerun task with same setting (Reactivate api)
- [x] Rerun task width different settings(This needs to delete it and recreate it to keep the same id)

### LINKED ISSUE #109: Detect task error and propose quick fix
When a task fail propose to fix with 
- Rerun
- Rerun with different params(Clone)

### PR DESCRIPTION
**Pull request by timotheeguerin**
fix #108 
fix #109

### CODE REVIEW

File: app/assets/styles/base/banner.scss [hunk 1]
```diff
@@ -1,112 +1,144 @@
...
 
         > .details-container {
-            padding: 10px 24px;
-            border-top: 1px solid $red-dark;
-            font-weight: 500;
+            border-top: 1px solid $accent;
         }
     }
+
+    .other-fixes {
```
Comment (paselem) [inline app/assets/styles/base/banner.scss]: What is the purpose of this class? Naming is a bit strange.

File: app/components/file/details/file-details.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { NgModule } from "@angular/core";
+
+import { commonModules } from "app/common";
+import { NodeBrowseModule } from "app/components/node/browse";
+import { PoolGraphsModule } from "app/components/pool/graphs";
```
Comment (paselem) [inline app/components/file/details/file-details.module.ts]: Why do we need the pools Graphs module here? Same goes for the start task module.

File: app/assets/styles/base/banner.scss (hunk 1 above)
Comment (timotheeguerin) [inline app/assets/styles/base/banner.scss]: In the banner there is the primary fix which is displayed directlry and then other fixes shown only after over

File: app/components/file/details/file-details.module.ts (hunk 2 above)
Comment (timotheeguerin) [inline app/components/file/details/file-details.module.ts]: yeah don't need those, might have been a copy and paste

**Pull request merged**